website/source/docs/acl/auth-methods/kubernetes.html.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: "docs"
+page_title: "Kubernetes Auth Method"
+sidebar_current: "docs-acl-auth-methods-kubernetes"
+description: |-
+  The kubernetes auth method type allows for a Kubernetes Service Account Token to be used to authenticate to Consul. This method of authentication makes it easy to introduce a Consul token into a Kubernetes Pod.
+---
+
+-> **1.5.0+:**  This guide only applies in Consul versions 1.5.0 and newer.
+
```

Comment:
kait: Can we add a brief overview that includes who the audience is, what is covered, and link to the auth method doc? If a user finds this page without reading the auth method doc first, they will belost so linking is good.